src/views/MyInfo.vue

```diff
@@ -0,0 +1,213 @@
+      <div class="column is-one-quarter">
+        <div class="box">
+          <h1 style="font-size: 1.2rem; font-weight: bold;">내가 차단한 유저 목록</h1>
+        </div>
+      </div>
+    </template>
+  </TheLayout>
+</template>
+
+<script>
+import store from '@/store'
+import { fetchArchives, fetchArticles, updateUser } from '@/api'
+import { fetchWithProgress } from './helper'
+import TheBoard from '@/components/TheBoard.vue'
+import TheLayout from '@/components/TheLayout.vue'
+
+export default {
+  name: 'my-info',
+  data () {
+    return {
+      nickname: null,
+      email: null,
+      picture: null,
+      pictureSrc: '',
+      sexual: false,
+      social: false,
+      updating: false,
+      archives: null,
+      articles: null
+    }
+  },
+  methods: {
+    // @TODO: setting update를 vuex에도 반영. 그냥 다시 fetch 해도 될지도 (...)
+    async updateSettings () {
+      this.updating = true
+      try {
+        await updateUser(store.getters.userId, {
+          nickname: this.nickname,
+          email: this.email,
+          picture: this.picture,
+          sexual: this.sexual,
+          social: this.social
+        }).then(res => {
+          store.commit('setUserProfile', res.data)
+        })
+      } catch (err) {
+        store.dispatch('error', '설정 변경 중 문제가 발생했습니다.')
+      }
+      this.updating = false
+    },
+    pictureHandler ({ target: { files: [ file ] } }) {
+      this.picture = file
+
+      const reader = new FileReader()
+      reader.onload = e => { this.pictureSrc = e.target.result }
+      reader.readAsDataURL(file)
+    }
+  },
+  async beforeRouteEnter (to, from, next) {
+    await fetchWithProgress([ store.dispatch('fetchMe') ])
+    const { userNickname, userEmail, userPicture, userConfig } = store.getters
+    next(vm => {
+      vm.nickname = userNickname
+      vm.email = userEmail
+      vm.pictureSrc = userPicture
+      vm.sexual = userConfig.sexual
+      vm.social = userConfig.social
+    })
+  },
+  components: { TheLayout, TheBoard },
+  async mounted () {
+    this.archives = await fetchArchives()
+    this.articles = await fetchArticles({ username: this.nickname })
+  }
+}
+</script>
+
+<i18n>
+ko:
+  profile: '프로필'
+  profile-photo: '프로필 사진'
+  nickname: '이름'
+  additional-settings: '추가 설정'
+  r-rated: '성인글 보기'
+  politics: '정치글 보기'
+  save: '저장'
+  logout: '로그아웃'
+en:
+  profile: 'My account'
+  profile-photo: 'Photo'
+  nickname: 'User name'
+  additional-settings: 'Settings'
+  r-rated: 'See adult contents'
+  politics: 'See political content'
+  save: 'Save'
+  logout: 'Logout'
+</i18n>
+
+<style lang="scss" scoped>
+.box {
+  padding: 40px;
+}
+
+.row {
+  display: flex;
+  align-items: center;
+}
+
+.profile-container {
+  display: inline-flex;
+  width: 6rem;
+  height: 6rem;
+  margin-bottom: 0.5rem;
+  background-color: grey;
+  border-radius: 50%;
+}
+
+.profile-image {
+  width: 6rem;
+  height: 6rem;
+  object-fit: cover;
+  border-radius: 50%;
+}
+
+.profile-change {
```

Comment:
This style seems useless. Please delete this if you don't have plan to use this.